squad/core/notification.py

```diff
@@ -203,6 +244,12 @@ def send_status_notification(status, project=None):
 
     notification.send()
 
+    threshold_metrics = status.get_exceeded_thresholds()
```

Comment:
IMHO threshold notifications should be send in the same message as regression/fix notifications. It's the same type of event just in context of measurement data. Is there a reason why the notification is separate?